### ISSUE TITLE
Implement a repository of models

### ISSUE THREAD

**Issue by mayou36** [zfit/zfit-developement#16]
@apuignav commented on [Fri May 04 2018](https://github.com/zfit/zfit/issues/17)

We should implement a system/repository to upload models once papers are published, sort of `pypi` for models. My idea is that it's a python package that uses a local version and if you don't have a local version it goes to a central server to grab the models.

I leave this here, but it's not urgent. We can start the discussion though, so we know exactly what to do.

---

@mayou36 commented on [Tue Oct 09 2018](https://github.com/zfit/zfit/issues/17#issuecomment-428148511)

We've been to a [TF Hub](https://www.tensorflow.org/hub/) talk, this seems to be what we looked for to store/share graphs (our models) easily.

In short: in the hub (cloud), a tf graph can be stored with a defined (by us) API on what is variable ("trainable"), what not and where input goes in/out. Loading/caching already implemented.

@rsilvaco is also in favor of that. Opinion, @apuignav ?

---

@apuignav commented on [Tue Oct 09 2018](https://github.com/zfit/zfit/issues/17#issuecomment-428163616)

Let's do it.
